src/Builder/Builder.php

```diff
@@ -18,14 +18,13 @@ public function __toString(): string
     /**
      * @param array<string|int, string> $mapping
      */
-    protected function buildQuery(array $mapping, Closure $callback, string $q = ''): string
+    protected function buildQuery(array $mapping, Closure $callback, string $query = ''): string
```

Comment:
I reckon there are other places where I use the `$q` variable, so just revert this so it's the same all over the lib :)

Reply:
Okay. I'll be the same cryptic `q`.

Reply:
Done.